aws/microservices/database-setup.yml

```diff
@@ -34,7 +34,7 @@ spec:
   containers:
   - name: tb-db-setup
     imagePullPolicy: Always
-    image: thingsboard/tb-node:3.4.1
+    image: thingsboard/tb-node:3.5.1
```

Comment:
this is concerning! I though we were fully using 3.5.1 since https://github.com/midokura/thingsboard-ce-k8s/pull/21

Reply:
Maybe there was a bump just in the node image, but not the chart. Maybe this is still something we should discuss, but I assume it's going to be better if we focus the dependency on the chart, rather than on the docker image.

cc @xavileon

Reply:
looking at everything again, the helm chart doesn't seem affected. the bumps are only in aws. azure, gcp, minikube, openshift folders. so maybe false alarm

Reply:
Yes, there is a mismatch between [chart](https://github.com/midokura/thingsboard-ce-k8s/blob/2a809d09eaf91e3e7d4fc53f53a9eb235a1d369b/helm/thingsboard/Chart.yaml#L22C1-L23C1) and (what is used for) [image](https://github.com/midokura/thingsboard-ce-k8s/blob/2a809d09eaf91e3e7d4fc53f53a9eb235a1d369b/helm/thingsboard/values.yaml#L20).

Reply:
I don't understand is why in [upstream/3.5.1 the tag for the image](https://github.com/thingsboard/thingsboard-ce-k8s/blob/release-3.5.1/helm/thingsboard/values.yaml#L20) is `3.4.0`. Is this an upstream error or intentional?

Reply:
It looks like an error, as other people have reported it: https://github.com/thingsboard/thingsboard-ce-k8s/issues/100

Reply:
> looking at everything again, the helm chart doesn't seem affected. the bumps are only in aws. azure, gcp, minikube, openshift folders. so maybe false alarm

yep, the bumps and changes in this PR we are not really using them, as we only use `helm` .